### ISSUE TITLE
How to set environment variables in code

### ISSUE THREAD

**Issue by yxing999** [DataDog/dd-trace-py#2501]
### Which version of dd-trace-py are you using?
0.49.2

### How can we reproduce your problem?
on 0.49.0 release note, I saw  

> The collect_metrics argument of the tracer.configure method has been deprecated. Runtime metrics should be enabled only via the DD_RUNTIME_METRICS_ENABLED environment variable.

It's not a good idea to set the environment variable for every applications, so I tried to set the `DD_RUNTIME_METRICS_ENABLED` in the library but it didn't work. looks like this environment variable was read before the library was loaded when using `ddtrace-run`[[link](https://github.com/DataDog/dd-trace-py/blob/2bde602f12a3fe61ddf898fc40d6e6f6bb6935a3/ddtrace/bootstrap/sitecustomize.py#L84)].  

For the case when we don't use `ddtrace-run`, the runtime metrics were not enabled as well.

### What is the result that you get?
No runtime metrics were sent.
### What is the result that you expected?
runtime metrics were sent.

**Comment by majorgreys** [comment]
@yxing999 You can enable runtime metrics in code as well:

```
from ddtrace.internal.runtime.runtime_metrics import RuntimeWorker
RuntimeWorker.enable()
```

**Comment by P403n1x87** [comment]
@yxing999 are there any logs that you could share with us? The `[INFO] Booting worker with pid: xxxx` message is generated by gunicorn and might be an indication of, e.g., a `SIGSEGV` and a continuous attempt to spawn worker processes.

**Comment by majorgreys** [comment]
This was resolved by #2564
